backend/controllers/storage.js

```diff
@@ -5,13 +5,19 @@ const config = require('../config');
 
 const separator = '_';
 
+function removeForbidenSymbols(str) {
+  const forbidenSymbolsRegExp = /[%\^!*'();:@&=+$,\/?#\[\]~><\s`\\|_]/g;
```

Comment:
Do we need to add  ' ' (space symbol) to regEx filter ?

Reply:
just to make sure it will work
this is not related to the name that user see